src/elements.jl

```diff
@@ -145,3 +145,20 @@ function local_potential_fourier(el::ElementCohenBergstresser, q::T) where {T <:
     qsq_pi = Int(round(q^2 / (2π / el.lattice_constant)^2, digits=2))
     T(get(el.V_sym, qsq_pi, 0.0))
 end
+
+"""
+Element for an ion with charge Z that generates a custom local potential.
+There is no check of the relation between pot_real and pot_fourier, the user
+has to ensure that point.
+"""
+struct ElementCustomIonPotential <: Element
+    Z::Int                  # Nuclear charge
+    pot_real::Function      # Real potential
+    pot_fourier::Function   # Fourier potential
+end
+charge_ionic(el::ElementCustomIonPotential) = el.Z
+
+"""1D custom Fourier potential: V(q) = int_{R^3} V(x) e^{-iqx} dx."""
+function local_potential_fourier(el::ElementCustomIonPotential, q::T) where {T <: Real}
+    return el.pot_fourier(q)
+end
```

Comment:
Since you're there, can you clean up the docstrings a bit? The docstring (eg for local_potential_real) should be docuumented once before the specific implementation for each type. You can do this with eg
```
"""
doc
"""
function X end
``` which defines a function but no methods.

Might as well mention that local_potential_radial is not currently used for anything other than debug/exploration